packages/opentelemetry-core/src/trace/NoopSpan.ts

```diff
@@ -16,14 +16,17 @@
 
 import * as types from '@opentelemetry/types';
 import { SpanContext } from '@opentelemetry/types';
+import { INVALID_SPAN_CONTEXT } from '../trace/spancontext-utils';
 
 /**
  * The NoopSpan is the default {@link Span} that is used when no Span
- * implementation is available. All operations are no-op except context
+ * implementation is available. All operations are no-op including context
  * propagation.
  */
 export class NoopSpan implements types.Span {
-  constructor(private readonly _spanContext: SpanContext) {}
+  constructor(
+    private readonly _spanContext: SpanContext = INVALID_SPAN_CONTEXT
```

Comment:
Why is the terminology different for the span and span context? (noop vs invalid)

Reply:
AFAIK no-op instance does do anything, no-op instance can be invalid but vice-versa is not true. Also, specs referred `SpanContext` as invalid with "0" value for `traceId` and `spanId`.